VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -60,14 +86,89 @@ def get_loop_devices(self):
                 disk_loop_devices_file_systems.append(file_system_info[0])
         return disk_loop_devices_file_systems
   
+    def disk_list_for_billing(self):
+        if(len(self.lsscsi_list) != 0):
+            for item in self.lsscsi_list:
+                idxOfColon = item.rindex(':',0,item.index(']'))# to get the index of last ':'
+                idxOfColon += 1
+                lunNumber = int(item[idxOfColon:item.index(']')])
+                # item_split is the list of elements present in the one row of the cmd sudo lsscsi
+                self.item_split = item.split()
+                #storing the corresponding device name from the list
+                self.device_name = self.item_split[len(self.item_split)-1]
```

Comment:
using a property defined on `self` to do the work of a local variable will leave `self` dirty and using memory that should be freed at the end of the loop. Please try and use a local variable for this, instead of `self.device_name`, that goes out of scope at the end of the loop.